src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs

```diff
@@ -82,10 +82,10 @@ public class CommandLineWrapper : ICommandLineWrapper
             if (redirectIO && streamOutputToInteractiveService)
             {
                 process.OutputDataReceived += (sender, e) => {
-                    _interactiveService.LogMessageLine(e.Data);
+                    _interactiveService.LogMessageLine(e.Data ?? string.Empty);
```

Comment:
Unrelated to current changes, `LogMessageLine` or `LogErrorLine` must accept the nullable strings.

Reply:
Fixed